.gitignore

```diff
@@ -1,6 +1,4 @@
+.vagrant
+# Not sure how relevant the below entries are still
 *.retry
 *.rsc
-locales/memphis/private_configs/extensions.conf.j2
-locales/memphis/private_configs/iax.conf.j2
-locales/memphis/private_configs/sip.conf.j2
-locales/memphis/private_configs/voicemail.conf.j2
```

Comment:
Unrelated to PR?

Reply:
Only related in that .gitignore had to get updated.  I'll be nuking the .retry and .rsc lines shortly, as I see Ansible no longer drops .retry by default, and .rsc (produced by mikrotik) isn't in the new world yet, and may never be.